src/components/timers/XY.tsx

```diff
@@ -1,3 +1,77 @@
-const XY = () => null;
+import React, { useState, useEffect } from 'react';
+
+const XY: React.FC = () => {
+  const initialTimePerRound = 60;
+  const totalRounds = 10;
+
+  // remaining time in current round
+  const [time, setTime] = useState(initialTimePerRound);
+  // current round
+  const [round, setRound] = useState(1);
+  // timer is running
+  const [isRunning, setIsRunning] = useState(false);
+
+ // useEffect handle timer interval when timer is running
+  useEffect(() => {
+    if (isRunning && time > 0) {
+      // decrement time by 1 sec
+      const interval = setInterval(() => {
+        setTime(prevTime => prevTime - 1);
+      }, 1000);
+
+      return () => clearInterval(interval);
+    } else if (isRunning && time === 0) {
+      if (round < totalRounds) {
+        setTime(initialTimePerRound); // next round timer reset
+        setRound(prevRound => prevRound + 1); // move to next round
+      } else {
+        setIsRunning(false);
+      }
+    }
```

Comment:
a potential issue here is if the component unmounts and you have an interval running. Since you would lose its handle, you won't be able to clear it. Use a ref to always have a handle to your intervals and clear them if the component unmounts.